src/plugins/lanplus/lanplus.c

```diff
@@ -1735,6 +1736,26 @@ ipmi_lanplus_build_v2x_msg(
 								msg + IPMI_LANPLUS_OFFSET_PAYLOAD, /* output */
 								&(payload->payload_length));       /* output */
 
+		if (old_payload_length != payload->payload_length)
+		{
+			len =
+				sizeof(rmcp)                +  // RMCP Header (4)
+				10                          +  // IPMI Session Header
+				2                           +  // Message length
+				payload->payload_length     +  // The actual payload
+				IPMI_MAX_INTEGRITY_PAD_SIZE +  // Integrity Pad
+				1                           +  // Pad Length
+				1                           +  // Next Header
+				IPMI_MAX_AUTH_CODE_SIZE;       // Authcode
+
+			uint8_t * new_msg = realloc(msg, len);
+			if (!new_msg) {
+				free(meg);
```

Comment:
You have a typo here (meg).

Reply:
Please verify that it builds for you before you push :)

Reply:
Apologize, I copied from another working directoriy and somehow missed that. I updated the PR (diff: https://github.com/ipmitool/ipmitool/compare/89fd59107c63fecea0cfa7b754714075db01083c..e9d837f810d79c6fed142bcf833e3b511052b302) and PTAL.